src/xsdba/_adjustment.py

```diff
@@ -541,12 +606,19 @@ def qm_adjust(
         The extrapolation method to use.
     kind : str
         The kind of correction to compute. See :py:func:`xsdba.utils.get_correction`.
+    adapt_freq_thresh : str, optional
+        Threshold for frequency adaptation. See :py:class:`xsdba.processing.adapt_freq` for details.
+        Default is None, meaning that frequency adaptation is not performed.
 
     Returns
     -------
     xr.Dataset
         The adjusted data.
     """
+    ds["sim"] = _adapt_freq_preprocess(
+        ds, adapt_freq_thresh, group=Grouper(group.name), dim=None
```

Comment:
```suggestion
        ds, adapt_freq_thresh, group=group, dim=None
```
I didn't catch that last time, but why are we redefining the group here ?

Reply:
In the case at hand, if we regroup with the window, it might be the case that `sim` will not be adapted on individual day-of-year after passing _adapt_freq and degrouping